vaadin-time-picker-flow-integration-tests/src/test/java/com/vaadin/flow/component/timepicker/tests/TimePickerLocalizationIT.java

```diff
@@ -121,13 +120,13 @@ public void testInitialValue_nonDefaultLocale_initialValueLocalizedCorrectly() {
 
     @Test
     public void testChangingStep_reduceStepToHigherScale_valueIsNotTooDetailed() {
-        runReduceStepTest(new Locale("en-US"), "4:00 PM", "PM");
-        runReduceStepTest(new Locale("en-CA"), "16:00", "p.m.");
-        runReduceStepTest(new Locale("fi-FI"), "16:00", "");
-        runReduceStepTest(new Locale("no-NO"), "16:00", "PM");
-        runReduceStepTest(new Locale("zh-TW"), "16:00", "");
-        runReduceStepTest(new Locale("ko-KR"), "16:00", "");
-        runReduceStepTest(new Locale("es-PA"), "16:00", "p. m.");
+        runReduceStepTest(new Locale("en", "US"), "4:00 PM", "PM");
+        runReduceStepTest(new Locale("en", "CA"), "16:00", "p.m.");
+        runReduceStepTest(new Locale("fi", "FI"), "16:00", "");
+        runReduceStepTest(new Locale("no", "NO"), "16:00", "PM");
+        runReduceStepTest(new Locale("zh", "TW"), "16:00", "");
+        runReduceStepTest(new Locale("ko", "KR"), "16:00", "");
+        runReduceStepTest(new Locale("es", "PA"), "16:00", "p. m.");
```

Comment:
These locales were previously incorrectly defined and the test just happened to work with the old implementation.

The first param is `language` and second is `country`. If you set language as "en-US" it's basically an invalid/undefined locale which returns "und" (undefined) if you do `locale.toLanguageTag()`.